scripts/zones/Abyssea-Attohwa/npcs/Red_Ghost.lua

```diff
@@ -0,0 +1,22 @@
+--------------------------------------------
+    -- NPC: Red Ghost
+    -- Zone: Abyssea - Attohwa
+    -- !pos 23 -0.744 126 215
+--------------------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    player:startEvent(389)
+end
+
+entity.onEventUpdate = function(player, csid, option, npc)    
+end
+
+entity.onEventFinish = function(player, csid, option, npc)
+    player:setPos(16, -5.644, 139, 175)
```

Comment:
Setting the position isn't needed, it's handled automatically by the cutscene and it's actually incorrect to set it since the player can actually decline the teleport after starting the cutscene. Same for Alrauverat in #3148.